google/cloud/bigquery_v2/types/standard_sql.py

```diff
@@ -89,7 +89,7 @@ class StandardSqlField(proto.Message):
 
     name = proto.Field(proto.STRING, number=1)
 
-    type = proto.Field(proto.MESSAGE, number=2, message=StandardSqlDataType,)
```

Comment:
This is a breaking change, but it's not a very commonly used interface (just for people who would be using the BQ Storage API directly)

Reply:
Actually, BQ Storage API reports schema in native serialization format (Avro / Arrow). This is a breaking change for folks that are introspecting the types of their BQML models.